packages/core/src/services/permission/permission.service.ts

```diff
@@ -14,139 +14,68 @@
+            permissionPoint.complete();
+            this._permissionPointMap.delete(permissionId);
         }
     };
 
-    addPermissionPoint = (unitID: string, item: PermissionPoint) => {
-        const permissionMap =
-            this._permissionPointMap.get(unitID) || new Map<string, BehaviorSubject<PermissionPoint>>();
-        if (!permissionMap.has(item.id)) {
-            permissionMap.set(item.id, new BehaviorSubject(item));
-            this._permissionPointMap.set(unitID, permissionMap);
-            return true;
+    addPermissionPoint = <T = boolean>(item: IPermissionPoint<T>) => {
+        if (!item.id) {
+            return false;
         }
-        return false;
+        this._permissionPointMap.set(item.id, new BehaviorSubject<IPermissionPoint<unknown>>(item));
+        this._permissionPointUpdate$.next(item);
+        return true;
     };
 
-    updatePermissionPoint = (unitID: string, permissionId: string, value: any) => {
-        const permissionMap = this._permissionPointMap.get(unitID);
-        if (!permissionMap) {
+    updatePermissionPoint = <T = boolean>(permissionId: string, value: T) => {
+        const permissionPoint = this._permissionPointMap.get(permissionId);
+        if (!permissionPoint) {
             return;
         }
-        const permissionSubject = permissionMap.get(permissionId);
-        if (permissionSubject) {
-            const subject = permissionSubject.getValue() as PermissionPoint;
-            subject.value = value;
-            subject.status = PermissionStatus.DONE;
-            permissionSubject.next(subject);
-        }
+        const subject = permissionPoint.getValue() as IPermissionPoint<T>;
+        subject.value = value;
+        subject.status = PermissionStatus.DONE;
+        permissionPoint.next({ ...subject });
+        this._permissionPointUpdate$.next(subject);
     };
 
-    getPermissionPoint = (unitID: string, permissionId: string) => {
-        const permissionMap = this._permissionPointMap.get(unitID);
-        if (!permissionMap) return;
-        const item = permissionMap.get(permissionId);
-        if (item) {
-            return item.getValue();
-        }
+    clearPermissionMap = () => {
+        this._permissionPointMap.clear();
     };
 
-    composePermission$(unitID: string, permissionIdList: string[]) {
-        const permissionMap = this._permissionPointMap.get(unitID);
+    getPermissionPoint = <T = boolean>(permissionId: string) => {
+        const permissionPoint = this._permissionPointMap.get(permissionId);
+        if (!permissionPoint) return;
+        return permissionPoint.getValue() as IPermissionPoint<T>;
+    };
+
+    getPermissionPoint$ = <T = boolean>(permissionId: string) => {
+        const permissionPoint = this._permissionPointMap.get(permissionId);
+        if (!permissionPoint) return;
+        return permissionPoint as Observable<IPermissionPoint<T>>;
+    };
+
+    composePermission$(permissionIdList: string[]) {
```

Comment:
Avoid use arrow functions. If you really need to bind `this`, do that in the constructor.